### PR TITLE
Register Subdomain: borris

### PR DESCRIPTION
**Pull request by linkb15**
Added the domain: borris

### CODE REVIEW

**Comment by WilliamDavidHarrison** [comment]
@linkb15 An A record only has numbers, for example: `8.8.8.8`

**Comment by WilliamDavidHarrison** [comment]
We don't support Netlify sorry.

**Comment by linkb15** [comment]
how about vercel?

**Comment by WilliamDavidHarrison** [comment]
> how about vercel?

No. We do support Railway though.

File: domains/borris.json [hunk 1]
```diff
@@ -0,0 +1,12 @@
+
+    {
+      "owner": {
+        "username": "linkb15",
+        "email": "borris.wiria@gmail.com"
+      },
+
+      "record": {
+        "CNAME": ["https://borris.netlify.app/"]
```
Comment (TheBotlyNoob) [inline domains/borris.json:9]: We don't support netlify

**Review by TheBotlyNoob — CHANGES_REQUESTED** [review]

**Pull request closed without merging**